app/queries/servers/post.ts

```diff
@@ -20,24 +20,17 @@ const {SERVER: {POST, POSTS_IN_CHANNEL, POSTS_IN_THREAD, PREFERENCE}} = MM_TABLE
 
 export const prepareDeletePost = async (post: PostModel): Promise<Model[]> => {
     const preparedModels: Model[] = [post.prepareDestroyPermanently()];
-    const relations: Array<Query<Model>> = [post.drafts, post.postsInThread];
-    for await (const relation of relations) {
+    const relations: Array<Query<Model>> = [post.drafts, post.postsInThread, post.files, post.reactions];
```

Comment:
Magic... the Query class has an implicit `then` so when you use await it returns the model instead of the query.. I spent hours trying to understand why TS was yelling at me

Reply:
And what if it crashes because the query doesn't work? (missing ids or whatever)
Or that cannot happen here?

Reply:
is a query that executes `fetch` so worse case scenario you get an empty array